src/request/sign-polygon-transaction/SignPolygonTransactionApi.js

```diff
@@ -137,9 +137,11 @@ class SignPolygonTransactionApi extends PolygonRequestParserMixin(TopLevelApi) {
             const targetAmount = /** @type {PolygonSwapDescription | PolygonSwapWithApprovalDescription} */ (description) // eslint-disable-line max-len
                 .args
                 .targetAmount;
-            if (targetAmount.lt(inputAmount.mul(99).div(100))) {
+            // Allow 1% slippage for swaps on Polygon mainnet, but up to 5% for testnet
+            const minTargetAmountPercentile = CONFIG.POLYGON_CHAIN_ID === 137 ? 99 : 95;
+            if (targetAmount.lt(inputAmount.mul(minTargetAmountPercentile).div(100))) {
```

Comment:
Not really a percentile.
Maybe like this?
```suggestion
            const maxInputAmountSlippagePercentage = CONFIG.POLYGON_CHAIN_ID === 137 ? 1 : 5;
            if (targetAmount.lt(inputAmount.mul(100 - maxInputAmountSlippagePercentage).div(100))) {
```